lib/geordi/commands/unit.rb

```diff
@@ -1,11 +1,23 @@
 desc 'unit', 'Run Test::Unit'
+long_desc <<-LONGDESC
+Runs Test::Unit with parallel_tests detection and `bundle exec`.
+
+In order to limit processes in a parallel run, you can set an environment
+variable like this: `PARALLEL_TEST_PROCESSORS=6 geordi unit`
```

Comment:
The LONGDESC will be rendered right below the short description in the README. Currently, it is a little redundant. What about:

```suggestion
Supports `parallel_tests`, binstubs and `bundle exec`.

In order to limit processes in a parallel run, you can set an environment
variable like this: `PARALLEL_TEST_PROCESSORS=6 geordi unit`
```